src/validation.cpp

```diff
@@ -1226,7 +1238,13 @@ MempoolAcceptResult MemPoolAccept::AcceptSingleTransaction(const CTransactionRef
 
     Workspace ws(ptx);
 
-    if (!PreChecks(args, ws)) return MempoolAcceptResult::Failure(ws.m_state);
+    if (!PreChecks(args, ws)) {
+        if (ws.m_state.GetResult() == TxValidationResult::TX_SINGLE_FAILURE) {
+            // Failed for fee reasons. It's helpful to know what the fee and vsize were.
```

Comment:
the chart was cleaned up, could these comments be fixed as well?

Reply:
Fixed the comment